shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -62,7 +61,7 @@ static BOOL DoesHardwareSupportWideGamut() {
   // 4. Settings from the main NSBundle and default values.
 
   NSBundle* mainBundle = FLTGetApplicationBundle();
-  NSBundle* engineBundle = [NSBundle bundleForClass:[FlutterViewController class]];
+  NSBundle* engineBundle = [NSBundle bundleForClass:[FlutterView class]];
```

Comment:
`FlutterViewController` was an arbitrary choice, it just needs to be a class in Flutter.framework. Swap to`FlutterView` since this is already migrated to ARC.

Reply:
can just use `FlutterDartProject` here?

Reply:
You're right, that's a better idea. 🙂